src/components/Categories.js

```diff
@@ -0,0 +1,5 @@
+const Categories = () => (
+  <h1>ikoote</h1>
+);
```

Comment:
- This component should have the `Button` element with the text: `check status`. Please the required changes.